modules/core/src/main/java/org/apache/ignite/internal/direct/stream/v2/DirectByteBufferStreamImplV2.java

```diff
@@ -271,10 +272,10 @@ public class DirectByteBufferStreamImplV2 implements DirectByteBufferStream {
     private int readItems;
 
     /** */
-    private Object[] objArr;
+    Object[] objArr;
```

Comment:
I propose to return the private access modifier, use `GridTestUtils#setFieldValue` in tests.